### PR TITLE
Make "extraKeys" consistent for notebook and editor

### PR DESCRIPTION
**Pull request by gnestor**
Closes https://github.com/jupyter/notebook/issues/2740

### CODE REVIEW

**Comment by Carreau** [comment]
I have no objections to changing the keymap, though that would be inconsistent with the notebook that default to "default" command amp. Would it be better to be consistent and change notebook as well ?

I also would try to get  a wider input on if we should change as it may break user habits.

**Comment by gnestor** [comment]
> Would it be better to be consistent and change notebook as well ?

It seems like sublime is the default for the notebook (e.g. if I <kbd>CMD + /</kbd> it will comment out a block).

**Comment by Carreau** [comment]
> It seems like sublime is the default for the notebook (e.g. if I CMD + / it will comment out a block).

Well no, the default is "default", the Ctrl-/ shortcut is set [here](https://github.com/jupyter/notebook/blob/f8d4d6ad2dcaaed68851483db1a4efe742b52278/notebook/static/notebook/js/cell.js#L139-L140)

**Comment by gnestor** [comment]
Ahhh! Ok, we don't need to make sublime the default, I just need to make the notebook and editor `extraKeys` consistent 👌

File: notebook/static/edit/js/editor.js [hunk 1]
```diff
@@ -76,12 +76,20 @@ function(
...
     Editor.default_codemirror_options = {
         extraKeys: {
-            "Tab" :  "indentMore",
+            "Cmd-Right": "goLineRight",
+            "End": "goLineRight",
+            "Cmd-Left": "goLineLeft",
+            "Tab": "indentMore",
+            "Shift-Tab" : "indentLess",
+            // "Cmd-Alt-[" : "indentAuto",
+            // "Ctrl-Alt-[" : "indentAuto",
```
Comment (takluyver) [inline notebook/static/edit/js/editor.js]: Should these be in or out? If we don't want them, let's remove them completely rather than commenting them.

Comment (gnestor) [reply notebook/static/edit/js/editor.js]: I'll remove...

Comment (takluyver) [reply notebook/static/edit/js/editor.js]: Looks like you've removed them from cell.js, but not here ;-)

**Comment by takluyver** [comment]
Reopening to rerun tests with master

**Pull request merged**